testdata/baselines/reference/submodule/conformance/jsxCheckJsxNoTypeArgumentsAllowed.js.diff

```diff
@@ -10,5 +10,5 @@
 -</>;
 +const component_1 = require("./component");
 +const React = require("react");
-+let x = (<component_1.MyComp />, <component_1.Prop> a={10} b="hi" />; // error, no type arguments in js
++let x = (<component_1.MyComp />, <Prop> a={10} b="hi" />; // error, no type arguments in js
```

Comment:
This is the only diff that isn't a strict improvement to match strada, imo - seems like swapping from the binder-based import-only resolver to the full emit resolver has changed the behavior of the cjs transform on the `Prop` identifier here. Now, this is all syntactic gobbeldeygook because it's a `jsx` file with a jsx tag with a type argument, being _generously_ reinterpreted as a comma expression, so the emit here doesn't actually matter, but might indicate a more important bug somewhere in the checker. It'd be trivial to swap the behavior back (without troubleshooting the root cause) by having both the import-only resolver and the checker-resolver on hand and using the import-only one for the cjs transform, though.